libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -44,7 +44,23 @@
   <typedef name="geomnamearray"/>
 
   <!-- ======================================================================== -->
-  <!-- Geometric Properties
+  <!-- Unit types                                                               -->
+  <!-- ======================================================================== -->
+   <unittypedef name="length" default="meter">
+      <unitdef name="nanometer" scale="1000000000.0" offset="0.0" />
+      <unitdef name="micron" scale="1000000.0" offset="0.0" />
+      <unitdef name="millimeter" scale="1000.0" offset="0.0" />
+      <unitdef name="centimeter" scale="100.0" offset="0.0" />
+      <!-- <unitdef name="meter" scale="1.0" offset="0.0" /> -->
```

Comment:
It is more convenient to have this defined to have a concrete default unit definition but is not necessary.
Wondering if this should be explicitly created on unittypedef Element. If it defined though it will create a duplicate element clash.

Reply:
I think it's better to leave out. Since the `default` attribute is required this would just be repeating the same information again, with increased risk for inconsistency and errors. The `scale` and `offset` for default must always be 1.0 and 0.0.